src/test/java/uk/gov/companieshouse/efs/api/paymentreports/service/PaymentReportServiceImplTest.java

```diff
@@ -235,6 +279,7 @@ void sendFinancePaymentFailureReportWhenEmpty() throws IOException {
 
         final List<PaymentReportEmailModel> values = emailCaptor.getAllValues();
 
+        assertThat(values.get(0).getHasNoPaymentTransactions(), is(true));
         assertThat(values.get(0).getFileLink(), is(successFileLink));
         assertThat(values.get(0).getFileName(), is(sh19ReportName.replace(".csv", "")));
         assertThat(values.get(1).getFileLink(), is(failedFileLink));
```

Comment:
For completeness add
`assertThat(values.get(1).getHasNoPaymentTransactions(), is(true));`

Reply:
ditto, added.